piet-gpu-hal/Cargo.toml

```
@@ -7,12 +7,12 @@ license = "MIT/Apache-2.0"
 edition = "2018"
 
 [dependencies]
-ash = "0.33"
-ash-window = "0.7"
-raw-window-handle = "0.3"
-bitflags = "1.2.1"
-smallvec = "1.6.1"
-bytemuck = "1.7.2"
+ash = "0.37"
+ash-window = "0.11"
+raw-window-handle = "0.5"
```

```suggestion
raw-window-handle = "0.4"
```
needs to match the dependencies

---

@m-hugo Which dependencies are using `0.4`? `ash-window`?

---

> @m-hugo Which dependencies are using `0.4`? ash-window 0.11?

yes